### PR TITLE
correct how we unstick styles in the beginning of the line

### PR DESCRIPTION
**Pull request by andrewfan**
Unstick styles for such situation

```
*bold*
|
```

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -209,13 +209,22 @@ const unstickyInlineStyles = (character, editorState) => {
...
   const block = content.getBlockForKey(selection.getStartKey());
+  const previouBlock = content.getBlockBefore(block.getKey());
   const entity = block.getEntityAt(startOffset - 1);
   if (entity !== null) return editorState;
 
   // If we're currently in a style, but the next character doesn't have a style (or doesn't exist)
   // we insert the characters manually without the inline style to "unsticky" them
-  const style = block.getInlineStyleAt(startOffset - 1);
-  if (style.size === 0) return editorState;
+  if (!startOffset && previouBlock) {
```
Comment (mxstbr) [inline src/index.js]: nit: `previous`

**Review by mxstbr — COMMENTED** [review]
Doesn't this already work as expected?



What am I missing, which case does this fix?

**Comment by andrewfan** [comment]
@mxstbr sorry, I did not specify the precondition
- create 2 empty lines
- focus the first line and type `*bold*`
- focus the second line, and type anything

**Comment by andrewfan** [comment]
@mxstbr, by the way, what tool do you use for screen recording? =)

**Review by mxstbr — APPROVED** [review]
Ahhh good catch, gotcha! 👍 Can repro, thanks for the PR!

**Comment by mxstbr** [comment]
I use https://getkap.co/, it's super awesome!

**Pull request merged**